packages/integration-tests/projects/suite-web/plugins/websocket-client.js

```diff
@@ -21,7 +21,7 @@ const createDeferred = id => {
     };
 };
 
-const DEFAULT_TIMEOUT = 20 * 1000;
+const DEFAULT_TIMEOUT = 60 * 1000;
```

Comment:
Is this used only for trezor-user-env? Maybe we could add a comment. Something in the sense that the timeout is quite high because tenv is synchronous and some commands take some time.

Reply:
Btw how long does the Shamir command take on your computer? Let's say 3 of 5 Shamir.

Reply:
Shamir 3 out of 5 (the 3 really does not play a role there in wallet creation process) took about 30 seconds.

Reply:
Ok let's see.